src/components/publiq-ui/Link.js

```diff
@@ -22,7 +22,7 @@ const Link = ({ href, children, as, className, ...props }) => {
 
   if (isInternalLink) {
     return (
-      <NextLink prefetch href={href} passHref>
+      <NextLink href={href} passHref>
```

Comment:
Why did you remove the prefetch here?

Reply:
I got console info that prefetch wasn't needed anymore

Reply:
Oh, nice. Probably new since 10 then.